cmd/vulcan-trivy/vulcan-trivy.go

```diff
@@ -118,14 +119,14 @@ func run(ctx context.Context, target string, optJSON string, state state.State)
 		"-f", "json",
 		"-o", reportOutputFile,
 	}
-	// Force vulnerability db cache update.
-	if opt.ForceUpdateDB {
+	// Skip vulnerability db update if not explicitly forced.
+	if !opt.ForceUpdateDB {
 		triviArgs = append(triviArgs, "--skip-update")
 	}
-	// Force skip update if image is latest and force update is not explicit.
+	// Log skip vulnerability db update if image is latest and force
+	// update is not explicit.
 	if !opt.ForceUpdateDB && strings.HasSuffix(target, "latest") {
```

Comment:
This if can be embedded in the previous one:
```
if skip {
  if latest {
  }
}
```

Reply:
Fixed.